examples/testapp/src/pages/payment/index.page.tsx

```diff
@@ -127,7 +127,8 @@ export default function Payment() {
       } else {
         toast({
           title: 'Payment failed',
-          description: 'error' in result ? result.error : 'Unknown error',
+          description:
+            'error' in result && typeof result.error === 'string' ? result.error : 'Unknown error',
```

Comment:
CI was broken, might be some disparity on CI and local, CI wasn't able to identify the discriminated union